MapleServer2/Types/Player.cs

```diff
@@ -94,6 +94,8 @@ public class Player
 
         public long PartyId;
 
+        public long ClubId;
```

Comment:
I saw discussion on this already but I just wanted to also put this here so we can have a record of it. Probably drop a TODO for this to be an array or something since players can be in 3 clubs.